docs/install/install-from-package/install-manager.rst

```diff
@@ -18,6 +18,17 @@ If you want to install a specific version:
 
    $ pip install -U backend.ai-manager==${BACKEND_PKG_VERSION}
 
+For the first time, you need to generate rpc keypair. This keypair is used for authentication and encryption to our manager-to-agent RPC connections.
+
+.. code-block:: console
+
+   $ cd "${HOME}/manager"
+   $ mkdir -p fixtures/manager
+   $ backend.ai mgr generate-rpc-keypair fixtures/manager 'manager'
```

Comment:
This is a minor one, but I prefer to generate the key pair under `fixtures/` not `fixtures/manager/`. It already is evident the `fixtures` will contain manager-only configurations here.

Reply:
I'll change `fixtures/manager` to `fixtures/`